src/cmd/cli/command/commands_test.go

```diff
@@ -140,84 +139,99 @@ func TestVersion(t *testing.T) {
 func TestCommandGates(t *testing.T) {
 	mockService := &mockFabricService{}
 	_, handler := defangv1connect.NewFabricControllerHandler(mockService)
+	origDir, err := os.Getwd()
+	if err != nil {
+		t.Fatalf("Failed to get current directory: %v", err)
+	}
+	composeDir := "../../../../src/testdata/sanity"
+	os.Chdir(composeDir)
 
 	server := httptest.NewServer(handler)
-	t.Cleanup(server.Close)
-
-	t.Setenv("AWS_REGION", "us-test-2")
+	t.Cleanup(func() {
+		os.Chdir(origDir)
+		server.Close()
+	})
 
-	if err := os.Chdir("../../../testdata/empty"); err != nil {
-		t.Fatalf("Failed to change directory: %v", err)
+	type cmdPermTest struct {
```

Comment:
It's common to not bother declaring these types and keep the inline since they are never reused